wgpu/_classes.py

```diff
@@ -457,25 +467,32 @@ def present(self):
         self._drop_texture()
         return result
 
-    def _present_bitmap(self):
-        texture = self._texture
-        device = texture._device
-
-        size = texture.size
-        format = texture.format
+    def _get_bytes_per_pixel(self, texture_format):
```

Comment:
This bit of the diff looks complicated, but all I did was move the logic that computes bytes_per_pixel in a method.